File: C  programs/Problems on Pattern Printing/Triangle Pattern/Pattern3.c
/* Accept the number of rows and number of columns from user  and Display below pattern.

Input:  iRow=4  iCol=4
Output : 1 2 3 4
         2 3 4
         3 4
	     4

*/
		  
#include<stdio.h>

void Pattern(int iRow,int iCol)
{
	int i=0,j=0;
	
	if(iRow<0)
	{
	   iRow=-iRow;
	}

	if(iCol<0)
	{
	   iCol=-iCol;
	}

	if(iRow !=iCol)
	{
	  printf("Invalid Input");
	  return;
	} 
	 
	for(i=1;i<=iRow;i++)  
	{
	   for(j=1;j<i;j++)   
	   {
		  printf(" ");
	   }
	   for(j=i;j<=iRow;j++) 
		{
		  printf("%d",j);  
	    }
		printf("\n");
     }
}

int main()
{
  int iValue1=0,iValue2=0;

  printf("Enter number of rows and columns");
  scanf("%d %d",&iValue1,&iValue2);

  Pattern(iValue1,iValue2);

  return 0;
}  	  

File: Java programs/Problems on Pattern Printing/Programs to print Alphabetical Square pattern/Pattern3.java
/*Write a java program which Accept the number of rows and number of columns from user and Display below pattern.

Input:  iRow=3   iCol=5
Output : A A A A A 
         B B B B B
         C C C C C
         D D D D D  

*/
		  
import java.util.*;

class Pattern
{
	public String str;
    public void Pattern(int iRow,int iCol)
    {
	   int i=0;
	   int j=0;
	   char ch='\0';
	   for(i=1,ch='A';i<=iRow;i++,ch++)
	   {  
           for(j=1;j<=iCol;j++)
	       {
			   System.out.print(ch+"\t");
		   }
	      System.out.println();
        }
	}
}

class Lba363
{
	public static void main(String arg[])
	{
		Scanner sobj=new Scanner(System.in);

		int iValue1=0,iValue2=0;

        System.out.println("Enter number of Rows: ");
        iValue1=sobj.nextInt();

		System.out.println("Enter number of Columns: ");
		iValue2=sobj.nextInt();

		Pattern pobj=new Pattern();

        pobj.Pattern(iValue1,iValue2);
	}	
}  	  